File: Example/ObjectiveC Example/Channel Plugin Sample objc/EntryViewController.m
//
//  EntryViewController.m
//  Channel Plugin Sample objc
//
//  Created by Haeun Chung on 21/03/2017.
//  Copyright © 2017 ZOYI. All rights reserved.
//

#import "EntryViewController.h"
#import "MainViewController.h"

@interface EntryViewController ()

@end

@implementation EntryViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
  if ([[segue identifier] isEqualToString:@"MainViewSegue"]) {
    MainViewController *viewController = [segue destinationViewController];
    viewController.isUser = false;
  }
}
@end

File: Example/Swift Example/Channel Plugin Sample/EntryViewController.swift
//
//  EntryViewController.swift
//  Example
//
//  Created by Haeun Chung on 14/03/2017.
//  Copyright © 2017 ZOYI. All rights reserved.
//

import Foundation
import UIKit

class EntryViewController : UIViewController {
  @IBOutlet var veilButton: UIButton!
  @IBOutlet var userButton: UIButton!
  
  override func viewDidLoad() {
    super.viewDidLoad()
    
    UIApplication.shared.statusBarStyle = .lightContent
  }
  
  override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
    if segue.identifier == "MainViewSegue" {
      if let destinationVC = segue.destination as? MainViewController {
        destinationVC.isUser = false
      }
    }
  }
}
